aquarius/app/util.py

```diff
@@ -59,12 +59,15 @@ def get_signature_vrs(raw):
         wallet = get_aquarius_wallet()
 
         keys_pk = keys.PrivateKey(wallet.key)
-        signed = keys.ecdsa_sign(message_hash=hashed_raw.digest(), private_key=keys_pk)
 
-        values = {
-            "hash": hashed_raw.hexdigest(),
-            "publicKey": wallet.address,
-        }
+        prefix = "\x19Ethereum Signed Message:\n32"
+        message = Web3.solidityKeccak(
+            ["bytes", "bytes"],
+            [Web3.toBytes(text=prefix), Web3.toBytes(hashed_raw.digest())],
+        )
+        signed = keys.ecdsa_sign(message_hash=message, private_key=keys_pk)
+
+        values = {"hash": hashed_raw.hexdigest(), "publicKey": wallet.address}
```

Comment:
Just a head's up: `hexdigest()` does not prepend `0x`.  I'm not sure if that's desired in this case but wanted to point it out nonetheless.

```pycon
>>> from hashlib import sha256
>>> msg = "hello world"
>>> sha256(msg.encode("utf-8")).hexdigest()
'b94d27b9934d3e08a52e52d7da7dabfac484efe37a5380ee9088f7ace2efcde9'
```